### DISCUSSION TITLE
Himalaya with Thunderbird

### DISCUSSION THREAD

**Discussion by lamyergeier** [pimalaya/himalaya#530] [Q&A]
I use thunderbird to store all my emails locally from Gmail using IMAP. Is it possible to make use of those already downloaded emails from thunderbird with Himalaya? This would prevent duplication of data.

**Comment by soywod** [comment]
If I remember well, Thunderbird stores email locally using custom structure. But it's possible to use instead the Maildir standard in your settings. With this option you should be able to configure a Maildir account and interract with it. Although if you plan to still use both Himalaya and Thunderbird it may not play well with cache.
